android/src/main/java/io/ltbl/bdkrn/BdkRnModule.kt

```diff
@@ -421,14 +485,17 @@ class BdkRnModule(reactContext: ReactApplicationContext) :
                     ReadableType.String -> {
                         resolvedIndex = (addressIndex as Dynamic).asString() ?: "new"
                     }
+
                     ReadableType.Number -> {
                         resolvedIndex = (addressIndex as Dynamic).asDouble() ?: "new"
                     }
+
                     else -> {
                         resolvedIndex = setAddressIndex("new")
                     }
                 }
-                val addressInfo = getWalletById(id).getInternalAddress(setAddressIndex(resolvedIndex))
+
+                val addressInfo = getWalletById(id).getAddress(setAddressIndex(resolvedIndex))
```

Comment:
Is this change on purpose? The ios/BdkRnModule.swift remains unchanged for this method.